Src/StartMenu/StartMenu.rc

```diff
@@ -81,7 +81,7 @@ BEGIN
             VALUE "FileDescription", "Open-Shell Menu"
             VALUE "FileVersion", _PRODUCT_VERSION_STR
             VALUE "InternalName", "Menu"
-            VALUE "LegalCopyright", "Copyright (C) 2017-2018, The Open-Shell Team"
+            VALUE "LegalCopyright", "Copyright (C) 2017-2048, MIT License, The Open-Shell Team"
```

Comment:
This is wrong.
There is no point to have year from the future in copyright message. It may even render copyright message invalid.

Usually there needs to be just single year in copyright notice, the year of first publication.
https://softwareengineering.stackexchange.com/questions/130478/what-copyright-date-for-an-update-to-an-open-source-project-from-last-year

I don't think there is any need to change copyright notices.
If we want to do it for some reason, then it should be done in all RC files, not just this one.

Reply:
well, then maybe without any annual notice, only MIT License @ge0rdi ?

Reply:
What is the issue with the copyright notice?
Why should license type be included in compiled binaries?

Is there some real issue you are trying to address?

Reply:
Copyright (C) 2017-2018

the copyright is in the past.. and do not fit.. imho

Reply:
Please, have a look at the article I've linked in previous comments.
Eventually google about how copyright notices work.

Reply:
close this commit ..